spirv_msl.cpp

```diff
@@ -13331,20 +13332,34 @@ void CompilerMSL::bitcast_from_builtin_load(uint32_t source_id, std::string &exp
 	case BuiltInBaseInstance:
 	case BuiltInBaseVertex:
 		expected_type = SPIRType::UInt;
+		expected_width = 4;
```

Comment:
Expression width in in bits, not bytes.

Reply:
And yet, miraculously, it didn't produce any changes in the tests...

Reply:
Ye, I guess it's because bitcasts tend to be implemented as value casts when the bit width matches. Changing sign is the only real use case for the builtin cast ...